### ISSUE TITLE
Couldn't find transformer version 5.0.0 that supports fall back support for manual attention implementation.

### ISSUE THREAD

**Issue by kzos** [huggingface/transformers#33858]
When I tried to add output_attentions=True, it says the work around in 'eager' can expose attention output, but its available in v5.0.0

`LlamaModel is using LlamaSdpaAttention, but `torch.nn.functional.scaled_dot_product_attention` does not support `output_attentions=True`. Falling back to the manual attention implementation, but specifying the manual implementation will be required from Transformers version v5.0.0 onwards. This warning can be removed using the argument `attn_implementation="eager"` when loading the model.
`Starting from v4.46, the `logits` model output will have the same type as the model (except at train time, where it will always be FP32)

How do i get v5.0.0 (when I check repo, I could find: release tags v4.5.1, dev note v4.6.x)?

**Comment by zucchini-nlp** [comment]
Hey @kzos! This is a FutureWarning and we don't have v5.0 released yet. It says that after releasing v5.0 we will no longer fallback to "eager" attention and you will have to `AutoModel.from_pretrained(model_id, attn_implementation="eager")` if you want to get attention scores in the output. Otherwise there is no need to indicate "eager" attention usually
